### PR TITLE
fix(checkins): fix patient checkin report

### LINKED ISSUE #770: (bug) Patient Checkin Report is broken
The Patient Checkin report no longer works via the button on the Patient Record page.  It is likely simply an outdated link or outdated report module.

**Steps to Reproduce**
1. Navigate to any patient record
2. Click the `Print` link on the Patient Checking box

**Expected Result**
1. A print dialog should appear with the PDF print link

**Actual Result**
1. A 404 error on `http://localhost:8080/reports/patients/:uuid/checkins?renderer=pdf`.

### PR DESCRIPTION
**Pull request by jniles**
This commit updates the patient checkin report to the latest report standard.  It adds integration tests for the patient checkin report route.

Closes #770.

---

Thank you for contributing!

Before submitting this pull request, please verify that you have:
- [x] Run your code through JSHint.  [Check out our styleguide](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/docs/STYLEGUIDE.md).
- [x] Run integration tests.
- [x] Run end-to-end tests.
- [x] Accurately described the changes your are making in this pull request.

For a more detailed checklist, [see the official review checklist](https://docs.google.com/document/d/1nupLVLRXgSZJQo_acLgrwvPnN8RukfSiwRhSToj81uU/pub) that this PR will be evaluated against.

Ensuring that the above checkboxes are completed will help speed the review process and help build a stronger application.  Thanks!

### CODE REVIEW

**Comment by jniles** [comment]
@mbayopanda, can I get a review?

File: server/controllers/medical/reports/checkins.js [hunk 1]
```diff
@@ -5,40 +5,24 @@
...
- * @requires path
  * @requires lodash
- * @requires BadRequest
- * @requires db
- * @requires renderers/json
- * @requires renderers/html
- * @requires renderers/pdf
+ * @requires lib/db
+ * @requires lib/ReportManager
+ * @requires Patients
```
Comment (mbayopanda) [inline server/controllers/medical/reports/checkins.js:11]: 👍

**Review by mbayopanda — COMMENTED** [review]
👍 All things in this PR are good, it just notice that the pdf report of checkin it is not translated.

Other thing is, the french transalation of `checkin` (verification) it is not well for the context, i think we have to choose a good name for example `Visite` and `Signaler visite du patient` for text on the checkin button.

File: server/controllers/medical/reports/checkins.js (hunk 1 above)
Comment (mbayopanda) [reply server/controllers/medical/reports/checkins.js:11]: Could we have a good name for `checkin` in the french transaction
chekin

`Verification des patients` is not very clear for the action to do.

**Comment by sfount** [comment]
I've created an issue to track the updates suggested by @mbayopanda https://github.com/IMA-WorldHealth/bhima-2.X/issues/811. 

I suggest that this pull request is merged with translations to follow.

**Pull request merged**